crates/modelardb_client/src/main.rs

```diff
@@ -272,38 +265,55 @@ fn execute_command(
         .next()
         .ok_or("no command was provided")?
     {
+        // Print the schema of a table on the server.
         "\\d" => {
-            //Print the schema of a table on the server.
             let table_name = command_and_argument
                 .next()
                 .ok_or("no table name was provided")?;
             let flight_descriptor = FlightDescriptor::new_path(vec![table_name.to_owned()]);
             let request = Request::new(flight_descriptor);
-            runtime.block_on(async {
-                let schema_result = flight_service_client
-                    .get_schema(request)
-                    .await?
-                    .into_inner();
-                let schema = convert::try_schema_from_ipc_buffer(&schema_result.schema)?;
-                for field in schema.fields() {
-                    print!("{}: {}", field.name(), field.data_type());
-                    for (metadata_name, metadata_value) in field.metadata() {
-                        print!(", {} {}", metadata_name, metadata_value);
-                    }
-                    println!();
+            let schema_result = flight_service_client
+                .get_schema(request)
+                .await?
+                .into_inner();
+            let schema = convert::try_schema_from_ipc_buffer(&schema_result.schema)?;
+            for field in schema.fields() {
+                print!("{}: {}", field.name(), field.data_type());
+                for (metadata_name, metadata_value) in field.metadata() {
+                    print!(", {} {}", metadata_name, metadata_value);
                 }
-                Ok(())
-            })
+                println!();
+            }
+            Ok(())
         }
+        // Print the name of the tables on the server.
         "\\dt" => {
-            //Print the name of the tables on the server.
-            if let Ok(tables) = retrieve_table_names(runtime, flight_service_client) {
+            if let Ok(tables) = retrieve_table_names(flight_service_client).await {
                 for table in tables {
                     println!("{table}");
                 }
             }
             Ok(())
         }
+        // Flushes all data the server currently has in memory to disk.
+        "\\f" => execute_action(flight_service_client, "FlushMemory", "").await,
+        // Flushes all data the server currently has in memory and disk to the object store.
+        "\\F" => execute_action(flight_service_client, "FlushEdge", "").await,
+        // Print helpful information, explanations with \\ must be indented more to be aligned.
+        "\\h" => {
+            println!(
+                "CREATE DDL        Execute a CREATE TABLE or CREATE MODEL TABLE statement.\n\
```

Comment:
is "CREATE DDL" enough in the help or should it say "CREATE [MODEL] TABLE tbl(col type, ...)" to provide more executable help?

Reply:
Fixed.